### DISCUSSION TITLE
Svelte documentation around `query` function

### DISCUSSION THREAD

**Discussion by srcrip** [urql-graphql/urql#2679] [Q&A] [answered]
Hi,

I'm a little confused about the documentation for Svelte around the `query` function. It's mentioned a lot in examples but it seems to be gone in V3. Is this a correct assessment or am I missing something?

Are there issues or a timeline to update these Svelte examples? Is the Svelte v3 stuff ready to go?

**Comment by JoviDeCroock** [comment, marked as answer]
I see it mentioned once in the [variables](https://formidable.com/open-source/urql/docs/basics/svelte/#variables) section, anywhere else you are seeing it? This is an error in the docs, the new API only leverages `queryStore`
